static/app/views/alerts/rules/issue/setupAlertIntegrationModal.tsx

```diff
@@ -1,16 +0,0 @@
-import {Fragment} from 'react';
-
-import type {ModalRenderProps} from 'sentry/actionCreators/modal';
-
-function SetupAlertIntegrationModal({Header, Body}: ModalRenderProps) {
```

Comment:
Would the old slack button open this modal? In this case since it now opens your new modal you need to check feature flag in `MessagingIntegrationModal` because the old button has access to it. Maybe I'm misunderstanding.

Reply:
The old slack button still directs to [docs](https://develop.sentry.dev/integrations/slack/), the new button that opens this modal is only returned when the user has the feature flag